eZ/Bundle/EzPublishCoreBundle/Imagine/VariationPurger/ImageFileVariationPurger.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Imagine\VariationPurger;
+
+use eZ\Bundle\EzPublishCoreBundle\Imagine\VariationPathGenerator;
+use eZ\Publish\Core\FieldType\Image\ImageStorage\Gateway as ImageStorageGateway;
+use eZ\Publish\Core\IO\Exception\BinaryFileNotFoundException;
+use eZ\Publish\Core\IO\IOServiceInterface;
+use eZ\Publish\SPI\Variation\VariationPurger;
+use Iterator;
+
+/**
+ * Purges image aliases based on image files referenced by the Image FieldType.
+ *
+ * It uses the FieldType's Storage Gateway.
```

Comment:
This description looks obsolete. It's not based on the FT storage gateway nor the FT itself.